integrations/cli/index.test.ts

```diff
@@ -1727,7 +1727,10 @@ test(
         }
         @supports (not (-webkit-appearance: -apple-pay-button))  or (contain-intrinsic-size: 1px) {
           ::placeholder {
-            color: color-mix(in oklab, currentColor 50%, transparent);
+            color: currentColor;
+            @supports (color: color-mix(in lab, red, red)) {
+              color: color-mix(in oklab, currentColor 50%, transparent);
+            }
           }
```

Comment:
This looks a bit silly because we already have an `@supports` query in preflight and now we chain another one but when you think about it, the first one only makes sure that Safari versions that crash are gated out. Older versions of Chrome/Firefox _that do not support `color-mix(…)` would still not have a default in that case so this is a small improvement maybe?

In the future we might be able to clean this up if we use a different pass for applying that polyfill that has access to the parent path and then we can bail if this is already inside an `@supprots` or so.